airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/NamingConventionTransformer.java

```diff
@@ -45,4 +47,8 @@
   @Deprecated
   String getTmpTableName(String name);
 
+  String getStageName(String schemaName, String tableName);
+
+  String getStagingPath(String connectionId, String schemaName, String tableName, DateTime writeDatetime);
```

Comment:
These methods do not look necessary for all destination. Should they be put in this interface?